VadeSecure/vade_cloud/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
```

Comment:
Could you add an entry in the changelog like [this one](https://github.com/SEKOIA-IO/intake-formats/blob/main/Darktrace/darktrace-threat-visualizer/CHANGELOG.md?plain=1#L10-L14)?